caas/kubernetes/clientconfig/k8s.go

```diff
@@ -162,3 +161,60 @@ func getKubeConfigPath() string {
 	}
 	return envPath
 }
+
+// getKubeConfig reads kubeconfig file either stdin or config file, then deserializes the contents into Config object.
+func getKubeConfig() (*clientcmdapi.Config, error) {
+	var filename string
+	var raw []byte
+
+	info, err := os.Stdin.Stat()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	if info.Mode()&os.ModeNamedPipe != 0 || info.Size() > 0 {
```

Comment:
I think there is some formatinng issues here. For e.g.,  I would have expected spaces between ```.Mode() & os.```

Reply:
here is not && but &